lib/spack/spack/concretize.py

```diff
@@ -701,7 +701,7 @@ def find_spec(spec, condition, default=None):
         visited.add(id(relative))
 
     # Then search all other relatives in the DAG *except* spec
-    for relative in spec.root.traverse(deptypes=all):
+    for relative in spec.root.traverse(deptype='all'):
```

Comment:
😨

Reply:
Turns out that this doesn't matter, since `deptype='all'` is the default value.

Reply:
it would be amazing if `deptype` tests for equivalence with the python builtins `any` and `all` 😂